internal/actor/donor.go

```diff
@@ -32,7 +33,7 @@ type Donor struct {
 }
 
 func (d Donor) FullName() string {
-	return fmt.Sprintf("%s %s", d.FirstNames, d.LastName)
+	return strings.Trim(fmt.Sprintf("%s %s", d.FirstNames, d.LastName), " ")
```

Comment:
It was to protect specifically against the view logic for `your_details` where we won't have a value yet and I'm doing a `not .App.DonorFullName`...even if I invert the logic we would still end up with a whitesapce string.

I guess we could trim as we set it in appData rather than here?

Reply:
ah, got you, that's cool then